packages/syft/src/syft/service/job/job_stash.py

```diff
@@ -92,7 +96,7 @@ class JobV2(SyftObject):
 
 
 @serializable()
-class Job(SyftObject):
+class JobV3(SyftObject):
```

Comment:
I think we don't need to create a new job class version. Since Version 3 is the latest one being edited in dev.

Reply:
Ah I see, fixed it here https://github.com/OpenMined/PySyft/pull/8407